html/components/highlight-board.stories.js

```diff
@@ -22,7 +22,7 @@ then they should be \`<button>\` elements.
 
 export const defaultStory = () => `
     <div
-     class="sprk-c-HighlightBoard sprk-c-HighlightBoard--has-image sprk-u-mbm"
```

Comment:
I just removed this util from html and from react. React actually had it inside the component. Angular doesnt have it at all. If people want it back, they can supply a spacing class to the additionalclasses for react of their own or just put something on it for html.